commit/plugin_e2e_test.go

```diff
@@ -324,6 +324,9 @@ func setupAllNodesReadAllChains(ctx context.Context, t *testing.T, lggr logger.L
 		n.ccipReader.On("NextSeqNum", ctx, []cciptypes.ChainSelector{chainA, chainB}).
 			Return([]cciptypes.SeqNum{10, 20}, nil)
 
+		// transmission phase root staleness check passes
+		n.ccipReader.On("NextSeqNum", ctx, []cciptypes.ChainSelector{chainB}).
+			Return([]cciptypes.SeqNum{20}, nil)
```

Comment:
Shouldn't it be  `21` to be valid?
```suggestion
			Return([]cciptypes.SeqNum{21}, nil)
```

Also any reason for not adding chainA?

```
		n.ccipReader.On("NextSeqNum", ctx, []cciptypes.ChainSelector{chainA}).
			Return([]cciptypes.SeqNum{11}, nil)
```

Reply:
if 21 was already committed we wouldn't be able to post 21

Reply:
the report of this test does not include chainA

Reply:
we have to rename the interface method name to ``LastCommittedSeqNums` not `NextSeqNums`. I'll do it as a followup